cmd/smee/main.go

```diff
@@ -255,6 +258,17 @@ func main() {
 		if err != nil {
 			panic(fmt.Errorf("failed to create backend: %w", err))
 		}
+		// Will print 10% of entries.
```

Comment:
has to be 2%?

Reply:
yeah, im actually going to remove the log line entirely. This was copy/paste from the `github.com/samber/slog-sampling` repo's example.